src/main/kotlin/com/sourcegraph/cody/agent/protocol/ClientCapabilities.kt

```diff
@@ -0,0 +1,13 @@
+package com.sourcegraph.cody.agent.protocol
+
+data class ClientCapabilities(
+    var completions: String? = null,
+    var chat: String? = null,
+    var git: String? = null,
+    var progressBars: String? = null,
+    var edit: String? = null,
+    var editWorkspace: String? = null,
+    var codeLenses: String? = null,
+    // 'lenses' for code lenses, must have codeLenses = 'enabled'
+    var fixupControls: String? = null,
```

Comment:
This is scree, I removed this and made the Agent always do client-side controls. (...which we can change if we want. But this is unused and should be removed without changes to add it back on the Agent side.)

Reply:
Removed.